.prettierrc.json

```
@@ -0,0 +1,3 @@
+{
+"@nhcarrigan/prettier-config"
+}
```

```suggestion
"@nhcarrigan/prettier-config"
```

Just needs to be a string here, not an object.